### PR TITLE
fix: "Confirm property change deletion"'s message on delete is only for property changes

### PR DESCRIPTION
**Pull request by nikmace**
Fix for #1530.

- Adds more descriptive text to the delete change dialog.

### CODE REVIEW

**Review by IvoSG — APPROVED** [review]
Code looks good
Test coverage is ok
Manual testing - done

**Review by IvoSG — APPROVED** [review]
Code looks good
Test coverage is ok
Manual testing - done

File: .changeset/curvy-rabbits-run.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux/control-property-editor': patch
+---
+
+"Confirm property change deletion"'s message on delete is only for property changes
```
Comment (zdravko-georgiev) [inline .changeset/curvy-rabbits-run.md:5]: Please add patch for `preview-middleware`.

**Review by zdravko-georgiev — CHANGES_REQUESTED** [review]

File: .changeset/curvy-rabbits-run.md (hunk 1 above)
Comment (GDamyanov) [reply .changeset/curvy-rabbits-run.md]: Done

**Review by voicis — APPROVED** [review]
- Small straight forward changes
- Good test coverage
- Tested locally

File: .changeset/curvy-rabbits-run.md (hunk 1 above)
Comment (zdravko-georgiev) [reply .changeset/curvy-rabbits-run.md]: Thx

**Review by zdravko-georgiev — APPROVED** [review]
Change looks good
Covered by unit test
Manual testing ✅

**Pull request merged**